### PR TITLE
Ck2 History Adaptation

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- `ck2-history-adaptation` becomes our `master`.

### CODE REVIEW

File: localization/english/culture/wc_cultures_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+ nathrezim:0 "Nathrezim"
+ nathrezim_collective_noun:1 "Nathrezim"
+ eredruin:0 "Ered'ruin"
+ eredruin_collective_noun:1 "Ered'ruin"
+ shivarra:0 "Annihilan"
+ shivarra_collective_noun:1 "Annihilans"
+ moarg:0 "Nathrezim"
+ moarg_collective_noun:1 "Nathrezim"
+ observer:0 "Ered'ruin"
+ observer_collective_noun:1 "Ered'ruin"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:106]: Why they are localized so?

File: localization/english/culture/wc_cultures_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+ dark_iron_collective_noun:1 "Dark Irons"
+ old_dwarven:0 "Dwarven"
+ old_dwarven_collective_noun:1 "Dwarves"
+ frostborn:0 "Frostborn"
+ frostborn_collective_noun:1 "Frostborn"
+
+ #Elven group
+ elf_group:0 "Elven"
+ elf_group_collective_noun:1 "Elves"
+ high_elf:0 "High Elf"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:127]: Back on CK2 it was...
```suggestion
 high_elf:0 "Quel'dorei"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+ high_elf_collective_noun:1 "High Elves"
+ blood_elf:0 "Blood Elf"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:129]: Back on CK2 it was...
```suggestion
 blood_elf:0 "Sin'dorei"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+ blood_elf_collective_noun:1 "Blood Elves"
+ night_elf:0 "Night Elf"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:131]: Back on CK2 it was...
```suggestion
 night_elf:0 "Darnassian"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+ night_elf_collective_noun:1 "Night Elves"
+ highborne:0 "Highborne"
+ highborne_collective_noun:1 "Highborne"
+ nightborne:0 "Nightborne"
+ nightborne_collective_noun:1 "Nightborne"
+ naga:0 "Naga"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:137]: Back on CK2 it was...
```suggestion
 naga:0 "Nazja"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 6]
```diff
@@ -0,0 +1,402 @@
...
+ scourge:0 "Scourge"
+ scourge_collective_noun:1 "Scourged"
+ forsaken:0 "Forsaken"
+ forsaken_collective_noun:1 "Forsaken"
+
+ #Ancient group
+ ancient_group:0 "Ancient"
+ ancient_group_collective_noun:1 "Ancients"
+ protector:0 "Protector"
+ protector_collective_noun:1 "Protectors"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:222]: It's their main culture.
```suggestion
 protector:0 "Ancient"
 protector_collective_noun:1 "Ancients"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 7]
```diff
@@ -0,0 +1,402 @@
...
+ tuskarr_group:0 "Tuskarr"
+ tuskarr_group_collective_noun:1 "Tuskarr"
+ tuskarr:0 "Tuskarr"
+ tuskarr_collective_noun:1 "Tuskarr"
+
+ #Shath'yar group
+ shathyar_group:0 "Shath'yar"
+ shathyar_group_collective_noun:1 "Shath'yar"
+ nraqi:0 "N'raqi"
+ nraqi_collective_noun:1 "N'raqi"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:298]: It's their main culture.
```suggestion
 nraqi:0 "Shath'yar"
 nraqi_collective_noun:1 "Old Ones"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 8]
```diff
@@ -0,0 +1,402 @@
...
+
+ #Tuskarr group
+ tuskarr_group:0 "Tuskarr"
+ tuskarr_group_collective_noun:1 "Tuskarr"
+ tuskarr:0 "Tuskarr"
+ tuskarr_collective_noun:1 "Tuskarr"
+
+ #Shath'yar group
+ shathyar_group:0 "Shath'yar"
+ shathyar_group_collective_noun:1 "Shath'yar"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:296]:
```suggestion
 shathyar_group_collective_noun:1 "Old Ones"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 9]
```diff
@@ -0,0 +1,402 @@
...
+ gnome_group:0 "Gnome"
+ gnome_group_collective_noun:1 "Gnomes"
+ gnome:0 "Gnome"
+ gnome_collective_noun:1 "Gnomes"
+
+ #Centaur group
+ centaur_group:0 "Centaur"
+ centaur_group_collective_noun:1 "Centaur"
+ centaur:0 "Centaur"
+ centaur_collective_noun:1 "Centaur"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:324]: Why it's not pluralized?
```suggestion
 centaur_group_collective_noun:1 "Centaurs"
 centaur:0 "Centaur"
 centaur_collective_noun:1 "Centaurs"
```

File: localization/english/culture/wc_cultures_l_english.yml [hunk 10]
```diff
@@ -0,0 +1,402 @@
...
+ jinyu:0 "Jinyu"
+ jinyu_collective_noun:1 "Jinyu"
+ ankoan:0 "Ankoan"
+ ankoan_collective_noun:1 "Ankoan"
+
+ #Quilboar group
+ quilboar_group:0 "Quilboar"
+ quilboar_group_collective_noun:1 "Quilboar"
+ quilboar:0 "Quilboar"
+ quilboar_collective_noun:1 "Quilboar"
```
Comment (zumbak04) [inline localization/english/culture/wc_cultures_l_english.yml:338]: Why it's not pluralized?
```suggestion
 quilboar_group_collective_noun:1 "Quilboars"
 quilboar:0 "Quilboar"
 quilboar_collective_noun:1 "Quilboars"
```

File: localization/english/culture/wc_cultures_l_english.yml (hunk 1 above)
Comment (Malicos) [reply localization/english/culture/wc_cultures_l_english.yml:106]: good catch

**Pull request merged**